stm32mp1.mk

```diff
@@ -70,6 +70,13 @@ STM32MP1_DTS_LINUX ?= $(STM32MP1_DTS_BASENAME)
 STM32MP1_DTS_U_BOOT ?= $(STM32MP1_DTS_BASENAME)
 STM32MP1_DEFCONFIG_U_BOOT ?= stm32mp15_defconfig
 
+# When enabled, WITH_STMM embeds StMM application in OP-TEE OS and default
+# enables WITH_RPMB_TEST for RPMB secure storage which StMM relies on.
+WITH_STMM ?= n
+ifeq ($(WITH_STMM),y)
+WITH_RPMB_TEST ?= y
```

Comment:
How about `WITH_RPMB_TEST ?= $(WITH_STMM)`? Doesn't that do the same thing as these three lines?

Reply:
Right. I'll update.